packages/api/src/routes/medical/internal-docs.ts

```diff
@@ -95,6 +98,61 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * POST /internal/docs/re-convert
+ *
+ * WARNING: This will remove all non-DocumentReferences from the FHIR server!
+ *
+ * Use the document references we have on FHIR server and the respective CDA on S3
+ * to re-convert them to FHIR and insert on the FHIR server.
+ *
+ * WARNING: This will remove all non-DocumentReferences from the FHIR server!
+ *
+ * Asychronous operation, returns 200 immediately.
+ *
+ * @param req.query.cxId - The customer's ID.
+ * @param req.query.patientIds - Optional comma-separated list of Patient IDs to filter document
```

Comment:
this is so dangerous if we accidentally miss it, that I would almost opt for making this param mandatory with at least 1 ID

as a side note, this list could be hundreds of patients, so should be make the patient/doc ID lists body params instead?

Reply:
We filter the doc refs by created date (from the doc ref mapping), so it will update only the patients from those doc refs - no risk to update all doc refs/resources of all patients.